src/components/dev-hub/icons/audio-player/fast-forward-icon.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FastForwardIcon = ({ width = '31px', height = '33px', ...props }) => (
+    <svg width={width} height={height} fill="none" {...props}>
```

Comment:
Can we add `title` tags for these SVGs? This helps to improve the accessibility (even if they are within a button) on reuse